components/fieldlabel/stories/template.js

```diff
@@ -9,6 +10,9 @@ export const Template = ({
   customClasses = [],
   size = "m",
   label,
+  alignment = "left",
+  isDisabled,
+  isRequired,
```

Comment:
```suggestion
  isRequired,
  style = { width: '72px' },
```

I'd suggest using the styleMap (need to add it as an import at the top) and setting this as the default - that way we can overwrite it if necessary when nesting.

Reply:
For styleMap this would need to be an array. Do I need to use styleMap for future nesting or do you think the object as you have it would be alright?

Reply:
StyleMap uses objects keyed on the property name so it wouldn't need to be an array. If you need to toggle properties on or off, the object approach is nice but it doesn't have to be done that way.